code/controllers/subsystems/job.dm

```diff
@@ -403,6 +403,17 @@
 
 	INVOKE_ASYNC(GLOBAL_PROC, .proc/show_location_blurb, H.client, 30)
 
+	if(SSticker.mode != "Extended" && joined_late)
+		var/antag_count = 0
+		for(var/antag_type in SSticker.mode.antag_tags)
+			var/list/cur_antags = get_antags(antag_type)
+			antag_count += length(cur_antags)
```

Comment:
```suggestion
		var/antag_count = 0
		for(var/antag_type in SSticker.mode.antag_tags)
			antag_count += antag_type.get_active_antag_count()
```

This helper might be more appropriate since it covers dead antags, SSD antags, destroyed antags?